hera_mc/cm_transfer.py

```diff
@@ -183,5 +183,7 @@ def __initialization(args):
                         for i, r in enumerate(row):
                             if r == '':
                                 r = None
+                            elif 'gpstime' in field_name[i]:
```

Comment:
Please add a comment here explaining why this is necessary — namely, the fact that the stoptime columns get written out as floats because they're nullable, and Pandas has to represent those null values as NaNs, and there's no such thing as a NaN integer.